packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -146,7 +147,11 @@ const renderApp = (
     ...rtl.render(
       <IntlProvider locale={locale}>
         <ApolloProviderComponent mocks={mocks} addTypename={addTypename}>
-          <ConfigureFlopFlip adapter={adpater} defaultFlags={flags}>
+          <ConfigureFlopFlip
+            adapter={adpater}
+            defaultFlags={flags}
+            adapterArgs={defaultFlopflipAdapterArgs}
```

Comment:
This didn't change in flopflip otherwise I wouldn've released a breaking change. I assume ESLint got better or something.